critiquebrainz/frontend/templates/review/entity/recording.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'review/entity/base.html' %}
+
+{% set recording = review.entity_id | entity_details(type='recording') %}
+
+{% block title %}
+  {% set recording_title = recording.name | default(_('[Unknown recording]')) %}
+  {{ _('Review of "%(recording)s" by %(user)s', recording=recording_title, user=review.user.display_name) }} - CritiqueBrainz
+{% endblock %}
+
+{% block entity_title %}
+  <h2 id="title">
+    {% if recording %}
+      <a href="{{ url_for('recording.entity', id=review.entity_id) }}">recording.name</a>
```

Comment:
```suggestion
      <a href="{{ url_for('recording.entity', id=review.entity_id) }}">{{ recording.name }}</a>
```

So it displays the recordings actual name!